macsio/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+# Header files need not be enumerated
+INCLUDE_DIRECTORIES(${CMAKE_CURRENT_SOURCE_DIR})
+
+# Source files
+SET(mio_srcs
+    macsio_clargs.c
+    macsio_mif.c
+    macsio_iface.c
+    macsio_timing.c
+    macsio_utils.c
+    macsio_log.c
+    macsio_data.c
+    macsio_work.c
+    macsio_main.c
+)
+
+SET_SOURCE_FILES_PROPERTIES(${mio_srcs} PROPERTIES LANGUAGE CXX)
```

Comment:
Elsa, any reason language was set to CXX here. AFAIK, this is all C code

Reply:
Ok, I recall now...it is because I am using non-constant static intialization in the plugins to set up the plugin maps at executable load time. But, I think that may mean that we only need C++ in the plugins, not the main code. I will investigate.